src/sorts/exchange_sort.jl

```diff
@@ -0,0 +1,12 @@
+function ExchangeSort!(arr::Vector{T})where T
```

Comment:
Missing documentation for this function and algorithm.

Reply:
Hi, thanks you giving me feedback, do I create a new PR by adding documentation?